src/SecureSign.Core/Signers/AuthenticodeSigner.cs

```diff
@@ -137,14 +142,58 @@ private async Task<Stream> SignUsingPowerShellAsync(string inputFile, string cer
 					"-command",
 					"\"$Cert = New-Object System.Security.Cryptography.X509Certificates.X509Certificate2;",
 					$"$Cert.Import('{CommandLineEncoder.Utils.EncodeArgText(certFile)}','{CommandLineEncoder.Utils.EncodeArgText(certPassword)}',[System.Security.Cryptography.X509Certificates.X509KeyStorageFlags]::DefaultKeySet);",
-					$"Set-AuthenticodeSignature '{CommandLineEncoder.Utils.EncodeArgText(inputFile)}' $Cert -Timestamp http://timestamp.digicert.com\"",
+					$"Set-AuthenticodeSignature '{CommandLineEncoder.Utils.EncodeArgText(inputFile)}' $Cert -Timestamp {_pathConfig.Timestamper}\"",
 				}
 			);
 
 			// PowerShell signs in-place, so just return the file we were given.
 			return File.OpenRead(inputFile);
 		}
 
+		/// <summary>
+		/// Signs the specified file using nuget.exe. Needs to be a nupkg
+		/// </summary>
+		/// <param name="inputFile">File to sign</param>
+		/// <param name="certFile">Path to the certificate to use for signing</param>
+		/// <param name="certPassword">Password for the certificate</param>
+		/// <returns>A signed copy of the file</returns>
+		private async Task<Stream> SignUsingNugetAsync(string inputFile, string certFile, string certPassword)
+		{
+			// if we aren't windows, we need to call nuget from mono
+			if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+			{
+				await RunProcessAsync(
+					"mono",
+					new[]
+					{
+						_pathConfig.Nuget,
+						"sign",
+						$"-CertificatePath \"{CommandLineEncoder.Utils.EncodeArgText(certFile)}\"",
+						$"-CertificatePassword \"{CommandLineEncoder.Utils.EncodeArgText(certPassword)}\"",
+						$"-Timestamper {_pathConfig.Timestamper}",
+						$"\"{CommandLineEncoder.Utils.EncodeArgText(inputFile)}\"",
+					}
+				);
+			}
+			else
+			{
+				await RunProcessAsync(
+					_pathConfig.Nuget,
+					new[]
+					{
+						"sign",
+						$"-CertificatePath \"{CommandLineEncoder.Utils.EncodeArgText(certFile)}\"",
+						$"-CertificatePassword \"{CommandLineEncoder.Utils.EncodeArgText(certPassword)}\"",
+						$"-Timestamper {_pathConfig.Timestamper}",
+						$"\"{CommandLineEncoder.Utils.EncodeArgText(inputFile)}\"",
```

Comment:
These arguments are all identical across both - Can you please remove the duplication (eg. pull them out into a separate array)?